package.json

```
@@ -5,13 +5,14 @@
   "author": "Qiskit Development Team",
   "license": "Apache-2.0",
   "scripts": {
-    "check": "npm run check:metadata && npm run check:spelling && npm run check:internal-links && npm run check:fmt",
+    "check": "npm run check:metadata && npm run check:orphan-pages && npm run check:spelling && npm run check:internal-links && npm run check:fmt",
```

Nit: we want to order these by what's most likely to fail for a _technical writer_:

```suggestion
    "check": "npm run check:metadata && npm run check:spelling && npm run check:internal-links && npm run check:orphan-pages && npm run check:fmt",
```